docs/admin/daemon.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: DaemonSet
+metadata:
+  name: prometheus-node-exporter
+spec:
+  template:
+    metadata:
+      name: prometheus-node-exporter
+      labels:
+        name: prometheus-node-exporter
```

Comment:
Please don't use "name" as a label key.
https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/config-best-practices.md

How about something like `daemon: monitoring`? If I were a cluster admin, I'd probably use the same label key for all my node agents / daemons, but not `name`. `name` is too generic (it could overlap with non-daemon pods) and confusing for users (due to similarity with metadata.name).